packages/amplify-schema-validator/src/exceptions/invalid-directive-error.ts

```diff
@@ -0,0 +1,7 @@
+export class InvalidDirectiveError extends Error {
```

Comment:
General comment here, but I'd like to see comments on methods and classes in the `src` directory. (excluding tests)

Reply:
Can we reuse the exception types from '@aws-amplify/graphql-transformer-core' package?

Reply:
Unfortunately, no. The intent here is not to pollute the dep chain w/ our FS and CDK dependencies so this can run in the Studio UI